### ISSUE TITLE
Link underline and state colors a11y

### ISSUE THREAD

**Issue by colinreedmiller** [vmware/clarity#5150]
Add underline to text link.
Check all colors for a11y against all BGs and update as necessary.

**Comment by colinreedmiller** [comment]
**All links should be underlined.**

Update colors as follows:

**Light theme**
Link - normal & hover use Blue (198) 800 --- HSL: (198, 100%, 27%)
• For Clarity NG use: Action-700
Link - clicked use Violet (282) 700 --- HSL: (282, 80%, 36%)
• For Clarity NG use: Secondary-Action-600
Link - visited use Lavender (238) 600 --- HSL: (238, 60%, 52%)

**Dark theme**
Link - normal & hover use Blue (198) 300 --- HSL:  (198, 100%, 70%)
Link - clicked use Violet (282) 200 --- HSL: (282, 73%, 83%)
Link - visited use Lavender (238) 100 --- HSL: (238, 58%, 90%)

(First value is color token e.g. Color / Hue / Value - second is HSL, NG variables provided where used)

![Links-5150](https://user-images.githubusercontent.com/44213553/97219261-f8134100-1786-11eb-8850-156b06612a3e.png)

**Comment by mathisscott** [comment]
@coryrylan 
Do we want to wrap this up in the core theming work?

**Comment by gnomeontherun** [comment]
Closing as part of the color theme work.
